server/manager/src/main/java/org/apache/accumulo/manager/Manager.java

```diff
@@ -1251,6 +1258,15 @@ boolean canSuspendTablets() {
       throw new IllegalStateException("Exception stopping status thread", e);
     }
 
+    tp.shutdown();
+    try {
+      final long rpcTimeout = getConfiguration().getTimeInMillis(Property.GENERAL_RPC_TIMEOUT);
+      tp.awaitTermination(Math.max(10000, rpcTimeout / 3), TimeUnit.MILLISECONDS);
+    } catch (InterruptedException e) {
+      log.debug("Interrupted while fetching status");
+    }
```

Comment:
Could probably only call shutdownNow here (assuming this is in the manager shutdown code)

```suggestion
```

Reply:
Removed this code in 967df31